python/ray/tests/test_gpu_objects_gloo.py

```diff
@@ -206,6 +217,77 @@ def test_p2p(ray_start_regular):
     assert ray.get(result) == pytest.approx(medium_tensor * 2)
 
 
+def test_p2p_errors_before_group_creation(ray_start_regular):
+    world_size = 2
+    actors = [GPUTestActor.remote() for _ in range(world_size)]
+
+    small_tensor = torch.randn((1,))
+    sender = actors[0]
+    receiver = actors[1]
+
+    with pytest.raises(
+        ValueError,
+        match="Actor.* does not have tensor transport GLOO available. Please create a communicator with `ray.experimental.collective.create_collective_group` before calling actor tasks with non-default tensor_transport.",
+    ):
+        ref = sender.echo.remote(small_tensor)
+
+
+@pytest.mark.parametrize("has_tensor_transport_method", [True, False])
+def test_p2p_blocking(ray_start_regular, has_tensor_transport_method):
```

Comment:
what's the relationship between this test and this PR?

Reply:
It's to check that the background concurrency group is created correctly no matter how the user specifies the tensor transport.